### DISCUSSION TITLE
Bug show search input, filter

### DISCUSSION THREAD

**Discussion by Khangdeptrai253** [rappasoft/laravel-livewire-tables#1844] [Feature Requests]
hello, i have problem with rappasoft/laravel-livewire-tables v3.4.5 when i run composer update
but it don't show (bellow picture)
![Screenshot from 2024-08-14 15-08-35](https://github.com/user-attachments/assets/bebc5519-c3fd-4620-9f2f-f58856c6d26e)

this is my code 
![Screenshot from 2024-08-14 15-11-02](https://github.com/user-attachments/assets/40c86c07-a223-4867-b569-e02101c9d78a)
![Screenshot from 2024-08-14 15-11-30](https://github.com/user-attachments/assets/3f3bdafe-3601-48b5-a5c2-3456b18fb7b9)
what can i do, to show search input and filter ?

**Comment by Khangdeptrai253** [comment]
please support me  :<

**Comment by annguyn2303** [comment]
yeah me too, pls fix this! @rappasoft

**Comment by Hungngyxn** [comment]
yah me too, i use version 3.4.5

**Comment by lrljoe** [comment]
@annguyn2303 @Hungngyxn  @Khangdeptrai253 
Couple of key questions:
- Please can you confirm if you have previously published the views?
and
- Please can you confirm if you are bundling the js/css, or injecting it?

Please also confirm:
- Which Theme you are using (Tailwind? Bootstrap-4? Bootstrap-5?)

**Comment by lrljoe** [comment]
I suggest looking at #1845 where the issue is explained a bit more!
